Phonebook.Frontend/src/app/shared/config/columnDefinitions.ts

```diff
@@ -222,6 +223,19 @@ export const ColumnDefinitions: {
       return Helpers.stringCompare(a.Business.Costcenter, b.Business.Costcenter) * Helpers.sortDirection(sortDirection);
     }
   },
+  type: {
```

Comment:
Can you please use a different name than `type` this is a reserved property in typescript (sometimes). Just to prevent confusion.

Reply:
what would be a good name?

Reply:
vielleich `group` oder `category` oder `person_type`

Reply:
Paul mean that "status" would be perfect because this is the same name like in the backend